code/game/machinery/recycler.dm

```diff
@@ -189,9 +189,9 @@
 	L.Paralyse(10 SECONDS)
 
 	// For admin fun, var edit emagged to 2.
-	if(gib || emagged == 2)
+	if(gib || emagged == 2) // emagged is usually treated as a boolean (except for bots), so this is a weird exception
```

Comment:
That does indeed sound pretty hacky.
I'd rather there was an extra `var/gib_always` var on the recycler than this..

Reply:
Doesn't solve bots having emagged == 2, but still seems like a good suggestion. So done.